front/src/app/components/admin/conventions/ConventionManageActions.tsx

```diff
@@ -187,15 +183,6 @@ export const ConventionManageActions = ({
           </VerificationActionButton>
         )}
 
-        {isRemindingAllowed(convention, roles) && (
```

Comment:
On le supprime également de la vue de pilotage convention (pour les prescripteurs) ?

Reply:
oui c'est ce qui est prévu

Reply:
de toute façon c'est juste un mail to